### PR TITLE
WM4 - Azin Yadegari - JavaScript Core2 - Week 3

### PR DESCRIPTION
**Pull request by AzinYad**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:Azin 
- Your City: Birmingham
- Your Slack Name: AzinYad

# Homework Details

- Module: JS Core2
- Week: 3

# Notes

- What did you find easy?
- exercises

- What did you find hard?
- project

- What do you still not understand?

- Any other notes?

-----
[View rendered 1-exercises/A-reading-list/README.md](https://github.com/AzinYad/JavaScript-Core-2-Coursework-Week3/blob/main/1-exercises/A-reading-list/README.md)
[View rendered 1-exercises/B-todo-list/README.md](https://github.com/AzinYad/JavaScript-Core-2-Coursework-Week3/blob/main/1-exercises/B-todo-list/README.md)

### CODE REVIEW

File: 2-projects/1-todo-list/script.js [hunk 1]
```diff
@@ -16,10 +49,42 @@ populateTodoList(todos);
...
+    for (let todo of allTodo) {
+      if (todo.style.textDecoration === "line-through") {
+        list.removeChild(todo);
+      } else {
+        todo.style.textDecoration = "none";
+      }
+    }
+  });
 }
+
```
Comment (Dawit-Dev) [inline 2-projects/1-todo-list/script.js:89]: Well done. I like how you came up with the solution.

File: 2-projects/1-todo-list/script.js [hunk 2]
```diff
@@ -1,6 +1,39 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  todos.forEach((todo) => {
+    const newLi = document.createElement("li");
+    const newLiSpan = document.createElement("span");
+    const newCheckIcon = document.createElement("i");
+    const newTrashIcon = document.createElement("i");
+
+    (newCheckIcon.ariaHidden = "true"), (newTrashIcon.ariaHidden = "true");
```
Comment (jdbevan) [inline 2-projects/1-todo-list/script.js:10]: Why did you write this line like this?

File: 2-projects/1-todo-list/script.js [hunk 3]
```diff
@@ -1,6 +1,39 @@
...
+
+    newCheckIcon.addEventListener("click", () => {
+      if (newLi.style.textDecoration === "none") {
+        newLi.style.textDecoration = "line-through";
+      } else {
+        newLi.style.textDecoration = "none";
+      }
+    });
+
+    newTrashIcon.addEventListener("click", () => list.removeChild(newLi));
```
Comment (jdbevan) [inline 2-projects/1-todo-list/script.js:35]: I like the way you grouped common/related code together like creating the elements and appending them and adding classes

**Review by jdbevan — COMMENTED** [review]
Great effort!

**Pull request closed without merging**